src/Components/Aspire.Microsoft.Azure.Cosmos/AspireMicrosoftAzureCosmosExtensions.cs

```diff
@@ -101,6 +243,8 @@ private static void AddAzureCosmosClient(
             });
         }
 
+        configureClientOptions?.Invoke(clientOptions);
```

Comment:
Why are we calling this here and on line 254?

Reply:
Missed this while refactoring the method. Fixed.